### ISSUE TITLE
Getting started with Speech Command Recognition With Tensorflow.JS and React.JS

### ISSUE THREAD

**Issue by Worawat101** [section-engineering-education/engineering-education#1693]
### Proposed title of the article
Getting started with Speech Command Recognition With Tensorflow.JS and React.JS 

### Introduction paragraph (2-3 paragraphs):
In this tutorial, we will learn how to perform speech command recognition in real-time and demonstrate the result of predictions. The idea is to create a React app and trigger the web browser system microphone that feeds the audio data. Then, we are going to load the speech-commands model provided by [TensorFlow.org](http://tensorflow.org/). Feeding the audio data to the model neural network, we are going to listen to the audio and make the predictions. Lastly, we are also going to improve the accuracy recognition by applying argmax function to our model.

### Key takeaways: 
- How to access the audio data stream from the browser microphone.
- How to predict the speech commands using a pre-trained speech-commands model from TensorFlow.
- How to improve the overall prediction/recognition probability by using the argmax function.

**Comment by ninjaginja** [comment]
good topic @Worawat101 - approved
